packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -0,0 +1,540 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  MutableRefObject,
+  ReactNode,
+  memo,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { Box } from "../Box";
+import { TagList } from "../TagList";
+import { Tag } from "../Tag";
+import { SearchField } from "../SearchField";
+import { Button } from "../Button";
+import { IconButton, Menu, Popover } from "@mui/material";
+import {
+  CheckIcon,
+  ChevronRightIcon,
+  CloseCircleFilledIcon,
+  FilterIcon,
+} from "../icons.generated";
+import { MenuItem } from "../MenuItem";
+import { Paragraph, Subordinate } from "../Typography";
+import { TextField } from "../TextField";
+import { CheckboxGroup } from "../CheckboxGroup";
+import { Checkbox } from "../Checkbox";
+import { RadioGroup } from "../RadioGroup";
+import { Radio } from "../Radio";
+
+export const dataFilterVariantValues = [
+  "autocomplete",
+  "checkbox",
+  "date",
+  "date-range",
+  "multi-select",
+  "range",
+  "range-slider",
+  "select",
+  "text",
+] as const;
```

Comment:
Not saying we change it, but based on the grammar here, shouldn't this be `"auto-complete"`? If so, I'm fine changing it. No reason to match the component name, but the component name implies this is a word.
```suggestion
export const dataFilterVariantValues = [
  "auto-complete",
  "checkbox",
  "date",
  "date-range",
  "multi-select",
  "range",
  "range-slider",
  "select",
  "text",
] as const;
```